src/unix/mod.rs

```diff
@@ -1615,7 +1615,7 @@ cfg_if! {
                 all(target_os = "freebsd", any(freebsd11, freebsd10)),
                 link_name = "readdir_r@FBSD_1.0"
             )]
-            #[allow(non_autolinks)] // FIXME: `<>` breaks line length limit.
+            #[allow(non_autolinks)] // FIXME(solaris): `<>` breaks line length limit.
```

Comment:
I guess this is a rustdoc thing?

```suggestion
            #[allow(non_autolinks)] // FIXME(docs): `<>` breaks line length limit.
```

Reply:
Oh ok, I can change it to `docs` for sure.